docs/data-sources/addons.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: "herokux"
+page_title: "Herokux: herokux_team"
+sidebar_current: "docs-herokux-datasource-addons-x"
+description: |-
+  Get list of add-ons installed on a specific App.
+---
+
+# Data Source: herokux_addons
+
+Use this data source to get a list of add-ons installed on a specific app.
+
+## Example Usage
+
+```hcl
+data "herokux_addons" "foobar" {
+  app_id = "MY_APP_ID"
+  addon_service_name = "MY_ADDON_NAME"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `app_id` - (Required) The app ID
+
+* `addon_service_name` - (Optional) Filter add-ons by service name (or include all add-ons if omitted). E.g. `heroku-postgresql`
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `addon_ids` - A list of the add-ons IDs installed on the app.
+
+* `addon_names` - A list of the add-ons names installed on the app.
```

Comment:
It would be better use a Map/hash instead of two separate attributes here and not rely on the indexes of `addon_ids` and `addon_names` to match up . I'm thinking of this sort of data structure:

```
{
  "<ADDON_UUID>": "<ADDON_NAME>"
...
}
```

With the current implementation, it would not be easily possible to determine the addon name for an addon id.

Reply:
Great suggestion! using a Map now